src/journal/views.py

```diff
@@ -1288,6 +1288,20 @@ def editorial_team(request, group_id=None):
 
     return render(request, template, context)
 
+def author_list(request):
+    """
+    Displays list of authors. 
+    :param request: HttpRequest object
+    :return: HttpResponse object
+    """
+    author_list = request.journal.users_with_role('author')
```

Comment:
There is a decorator `@has_journal` that can handle this also...

Reply:
`@has_journal` should suffice for this. Removing the check from the template for now is the way to go because the `users_with_role` function is specific to the journal being used already.